handler/core/src/main/java/io/knotx/fragments/task/TaskFactoryOptions.java

```diff
@@ -40,19 +44,14 @@ public JsonObject toJson() {
     return json;
   }
 
-  /**
-   * Gets {@link TaskFactory} name
-   *
-   * @return task provider factory name
-   */
   public String getFactory() {
     return factory;
   }
 
   /**
-   * Sets task provider factory name
+   * The task factory name that identifies {@code TaskFactory} implementation.
    *
-   * @param factory - task provider factory name
+   * @param factory - task factory name
    * @return reference to this, so the API can be used fluently
    */
   public TaskFactoryOptions setFactory(String factory) {
```

Comment:
We have convention (or at least majority of setters have) of fluent setters.
I'm not sure we need that in this particular case.

Reply:
The fluent API is useful in tests.

Reply:
I ment that this model is missing fluent API (setter does not return `this`).